### ISSUE TITLE
Separator request

### ISSUE THREAD

**Issue by tonypachino** [QuiteRSS/quiterss#1410]
Can you make a separator so that i can put it between feeds, i want to separate them into the categories?

For exampe like in Firefox bookmarks, right click on feed and choose: New Separator

**Comment by darcosion** [comment]
In the add-feed menu, there is a "folder" option, so you can create folder and put feed on folder, is that response to your issue ?

**Comment by tonypachino** [comment]
> 
> 
> In the add-feed menu, there is a "folder" option, so you can create folder and put feed on folder, is that response to your issue ?

No, go to the Firefox and try what i explained and you will know what i'm talking about.
